assets/scss/_variables_project.scss

```diff
@@ -0,0 +1,2 @@
+$primary: #326ce5;
```

Comment:
I'm surprised to see this change as part of the PR; consider rebasing against _main_?

Reply:
See https://www.docsy.dev/docs/adding-content/lookandfeel/#project-style-files

Reply:
Ah, that explains it. Using `_variables_project.scss` is correct.